Lib9c/Battle/RaidSimulator.cs

```diff
@@ -19,7 +19,6 @@ public class RaidSimulator : Simulator
         public int DamageDealt { get; private set; }
         public List<FungibleAssetValue> AssetReward { get; private set; } = new List<FungibleAssetValue>();
         public override IEnumerable<ItemBase> Reward => new List<ItemBase>();
-        private const int TurnLimit = 150;
```

Comment:
missing copy params in `RaidSimulatorV1`

Reply:
This field is not used so I removed it.